src/test/kotlin/no/digdir/informasjonsforvaltning/fdk_dataset_harvester/harvester/HarvesterTest.kt

```diff
@@ -237,4 +237,60 @@ class HarvesterTest {
         assertEquals(expectedReport, report)
     }
 
-}
\ No newline at end of file
+    @Test
+    fun ableToHarvestDatasetSeries() {
+        whenever(adapter.getDatasets(TEST_HARVEST_SOURCE_4))
+            .thenReturn(responseReader.readFile("harvest_response_4.ttl"))
+
+        whenever(valuesMock.catalogUri)
+            .thenReturn("http://localhost:5000/catalogs")
+        whenever(valuesMock.datasetUri)
+            .thenReturn("http://localhost:5000/datasets")
+
+        val report = harvester.harvestDatasetCatalog(TEST_HARVEST_SOURCE_4, TEST_HARVEST_DATE)
+
+        argumentCaptor<Model, String>().apply {
+            verify(turtleService, times(1)).saveAsHarvestSource(first.capture(), second.capture())
+            Assertions.assertTrue(first.firstValue.isIsomorphicWith(responseReader.parseFile("harvest_response_4.ttl", "TURTLE")))
+            Assertions.assertEquals(TEST_HARVEST_SOURCE_4.url, second.firstValue)
+        }
+
+        argumentCaptor<Model, String, Boolean>().apply {
+            verify(turtleService, times(1)).saveAsCatalog(first.capture(), second.capture(), third.capture())
+            Assertions.assertTrue(first.allValues[0].isIsomorphicWith(responseReader.parseFile("harvest_response_4.ttl", "TURTLE")))
+            assertEquals(listOf(CATALOG_ID_4), second.allValues)
+            Assertions.assertEquals(listOf(false), third.allValues)
+        }
+
+        argumentCaptor<CatalogMeta>().apply {
```

Comment:
Er denne nødvendig, siden du ikke sjekker capture?

Reply:
Den sjekker at det er 1 katalog som blir lagret, tenkte det var litt unødvendig å sjekke alle detaljene rundt den lagringen, siden det allerede blir testet. Men det at det er korrekt mengde kataloger som lagres er en fin indikasjon på at alt fungerer som normalt.

Men veldig nødvendig er den strengt tatt ikke...

Reply:
Skjønner, bare for å sjekke at den ble kalt antall ganger. Kan man da forenkle f.eks. slik:

verify(catalogRepository, times(1)).save(any())

På den måten ikke bry seg om parametre siden man ikke sjekker videre på det?

Reply:
slikleis? 7399929

Reply:
Ja, og kanskje man da også kan fjerne argumentCaptor som står rundt "verify".

Reply:
Tror det er argumentCaptor som teller hvor mange ganger save() osv blir kalt, dvs at slik de verify'ene er satt opp vil de ikke fungere utenfor argumentCaptor. Men jeg kan jo prøve, se hva jeg får til...

Reply:
TIL: Det er fullt mulig å kjøre verify med times() utenfor argumentCaptor :+1: 

Kjørte bare en amend på forrige refaktorering: a31fc46